### ISSUE TITLE
`git pull upstream master` instructions are broken

### ISSUE THREAD

**Issue by kevin-keraudren** [scikit-image/scikit-image#2071]
If we follow the instructions on http://scikit-image.org/docs/stable/contribute.html we have:

```
$ git remote add upstream git@github.com:scikit-image/scikit-image.git
$ git pull upstream master
Warning: Permanently added the RSA host key for IP address '192.30.252.121' to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

If instead we do:

```
$ git remote add upstream https://github.com/scikit-image/scikit-image.git
```

then `git pull upstream master` works fine.

I suggest updating the contribute page, unless I am the only one experiencing this, or maybe I am missing another undocumented step?

**Comment by stefanv** [comment]
You have to set GitHub up to know your SSH key. If that is not mentioned,
we should add a link.

**Comment by soupault** [comment]
@kevin-keraudren Hi, Kevin!
Yes, I agree, the case in the contributing guide will work only if you have already configured SSH keys for your GitHub account.
Could be nice to update the guide!

**Comment by JDWarner** [comment]
For the record, it's far easier and clearer for new contributors to use the HTTPS method originally proposed than force them to wade through SSH. Even if this means they have to type their password every time.

I strongly favor that approach. Have the SSH method an optional thing for advanced users if we must, but IMO it is inappropriate in a getting started guide.

**Comment by emmanuelle** [comment]
closed by #2073
